This allows Motif and Athena to override the symbols even when linking for
two-level namespace.
--- src/Vendor.c.orig	2023-04-10 00:48:48
+++ src/Vendor.c	2024-05-22 23:24:02
@@ -82,6 +82,9 @@
 #endif
 
 /* *INDENT-OFF* */
+#if defined(__APPLE__)
+__attribute__((weak))
+#endif
 externaldef(vendorshellclassrec)
 VendorShellClassRec vendorShellClassRec = {
     {
@@ -138,6 +141,9 @@
 /* *INDENT-ON* */
 
 #if !defined(AIXSHLIB) || !defined(SHAREDCODE)
+#if defined(__APPLE__)
+__attribute__((weak))
+#endif
 externaldef(vendorshellwidgetclass)
 WidgetClass vendorShellWidgetClass = (WidgetClass) (&vendorShellClassRec);
 #endif
